src/Desktop/UIAutomation/TreeWalkers/TreeWalkerForTest.cs

```diff
@@ -57,15 +58,17 @@ public TreeWalkerForTest(A11yElement element, BoundedCounter elementCounter)
         /// Refresh tree node data with all children at once.
         /// <param name="mode">indicate the mode</param>
         /// </summary>
-        public void RefreshTreeData(TreeViewMode mode)
+        public void RefreshTreeData(TreeViewMode mode, CancellationToken? cancellationToken = null)
```

Comment:
How do you feel about using `CancellationToken.None` instead of a nullable object?

Reply:
Good idea! I'll make that change.